apps/condo/migrations/20240419152456-0388_auto_20240419_1025.js

```diff
@@ -0,0 +1,156 @@
+CREATE INDEX "DocumentCategory_updatedAt_3f3516cb" ON "DocumentCategory" ("updatedAt");
+CREATE INDEX "DocumentCategory_deletedAt_969e3b65" ON "DocumentCategory" ("deletedAt");
+CREATE INDEX "DocumentCategory_createdBy_4e48136e" ON "DocumentCategory" ("createdBy");
+CREATE INDEX "DocumentCategory_updatedBy_0d429d68" ON "DocumentCategory" ("updatedBy");
+ALTER TABLE "Document" ADD CONSTRAINT "Document_category_760d72ff_fk_DocumentCategory_id" FOREIGN KEY ("category") REFERENCES "DocumentCategory" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "Document" ADD CONSTRAINT "Document_createdBy_57173a26_fk_User_id" FOREIGN KEY ("createdBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "Document" ADD CONSTRAINT "Document_organization_ba7c6576_fk_Organization_id" FOREIGN KEY ("organization") REFERENCES "Organization" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "Document" ADD CONSTRAINT "Document_property_fb12a0cf_fk_Property_id" FOREIGN KEY ("property") REFERENCES "Property" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "Document" ADD CONSTRAINT "Document_updatedBy_ec6bc064_fk_User_id" FOREIGN KEY ("updatedBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+CREATE INDEX "Document_createdAt_3cde854e" ON "Document" ("createdAt");
+CREATE INDEX "Document_updatedAt_899088c5" ON "Document" ("updatedAt");
+CREATE INDEX "Document_deletedAt_ab2f8c95" ON "Document" ("deletedAt");
+CREATE INDEX "Document_category_760d72ff" ON "Document" ("category");
+CREATE INDEX "Document_createdBy_57173a26" ON "Document" ("createdBy");
+CREATE INDEX "Document_organization_ba7c6576" ON "Document" ("organization");
+CREATE INDEX "Document_property_fb12a0cf" ON "Document" ("property");
+CREATE INDEX "Document_updatedBy_ec6bc064" ON "Document" ("updatedBy");
+
+---
+--- [CUSTOM] Set default values "canReadDocuments" to existed roles (all roles)
+---
+
+UPDATE "OrganizationEmployeeRole"
+SET "canReadDocuments" = true
+WHERE "name" = 'employee.role.Administrator.name'
+    OR "name" = 'employee.role.Dispatcher.name'
+    OR "name" = 'employee.role.Manager.name'
+    OR "name" = 'employee.role.Foreman.name'
+    OR "name" = 'employee.role.Technician.name'
+    OR "name" = 'employee.role.Contractor.name';
+    
+---
+--- [CUSTOM] Set default values "canManageDocuments" to existed roles (all without Contractor)
+---
+
+UPDATE "OrganizationEmployeeRole"
+SET "canManageDocuments" = true
+WHERE "name" = 'employee.role.Administrator.name'
+    OR "name" = 'employee.role.Dispatcher.name'
+    OR "name" = 'employee.role.Manager.name'
+    OR "name" = 'employee.role.Foreman.name'
+    OR "name" = 'employee.role.Technician.name';
+
+INSERT INTO "DocumentCategory" (name, id, v, "createdAt", "updatedAt", "deletedAt", "newId", dv, sender, "createdBy", "updatedBy") VALUES ('document.category.DefectActs.name', gen_random_uuid(), 1, now(), now(), null, null, 1, '{"dv": 1, "fingerprint": "migration-initial-create"}', null, null);
```

Comment:
Hey, lets generate IDs explicitly here, so that we can reference them in code later. Same trick was done in billing categories

Reply:
`INSERT INTO "DocumentCategory" (name, id, v, "createdAt", "updatedAt", "deletedAt", "newId", dv, sender, "createdBy", "updatedBy") VALUES ('document.category.DefectActs.name', '92c2415b-27a2-41ef-ad6b-ed4610a45446', 1, now(), now(), null, null, 1, '{"dv": 1, "fingerprint": "migration-initial-create"}', null, null);`